automation/build-artifacts.sh

```diff
@@ -17,7 +17,7 @@ rpmbuild \
     -D "_sourcedir $PWD/" \
     -D "_topmdir $PWD/rpmbuild" \
     -bs ./python-ovirt-engine-sdk4.spec
-
+<< ////
```

Comment:
Hmm, this is the 1st time I've heard about multiline comments in bash :-)
Personally adding `#` at beginning of each line seems more readable to me

Anyway please add a comment that below lines are not needed as Python 3.8 is not used at the moment